### ISSUE TITLE
Error on migrations: ModuleNotFoundError: No module named 'OpenSSL.rand'

### ISSUE THREAD

**Issue by guinslym** [Seedstars/django-react-redux-base#87]
On the installation, when I ran `python manage.py migrate` I get this following error

```
  File "/home/guinslym/Documents/Github/redux-tutorial/django-react-redux-base/env/lib/python3.6/site-packages/knox/models.py", line 5, in <module>
    from knox import crypto
  File "/home/guinslym/Documents/Github/redux-tutorial/django-react-redux-base/env/lib/python3.6/site-packages/knox/cryptpip install -U django-rest-knoxo.py", line 5, in <module>
    from OpenSSL.rand import bytes as generate_bytes
ModuleNotFoundError: No module named 'OpenSSL.rand'
```
I did '`pip install -U django-rest-knox'` but the error is still there. any help?

Thanks

**Comment by guinslym** [comment]
proposed fix by @philngo currently on a PR

`https://github.com/James1345/django-rest-knox/pull/69/commits/ade69178edac1d4d73dedee0ad933d4106e22c82`

**Comment by lfrodrigues** [comment]
Thanks @guinslym for the report and fix ;)  This is python 3.6, right? 
Closing issue as it's not related to this repo. We'll anyway update django-rest-knox once they merge fix.
